content/en/docs/tasks/administer-cluster/safely-drain-node.md

```diff
@@ -88,10 +88,10 @@ respect the PodDisruptionBudget you specify.
 For example, if you have a StatefulSet with three replicas and have
 set a PodDisruptionBudget for that set specifying `minAvailable: 2`,
 `kubectl drain` only evicts a pod from the StatefulSet if all three
-replicas pods are ready; if then you issue multiple drain commands in
+replicas pods are healthy; if then you issue multiple drain commands in
```

Comment:
Nit: maybe a link to what it means healthy for PDB.

Reply:
+1, made a link

- I have put links for both healthy mentions here since it is an external document
- in the PDB doc I just put a link to the healthiness section when mentioning the healthy first time, since we define the healthy later. But kept other instances without links to prevent too much needless cross referencing in the same doc.